aten/src/ATen/core/ivalue.cpp

```diff
@@ -96,6 +96,8 @@ TypePtr IValue::type() const {
       return toTuple()->type();
     case Tag::Generator:
       return GeneratorType::get();
+    case Tag::Enum:
+      TORCH_INTERNAL_ASSERT(false, "To be implemented");
```

Comment:
you should be able to get the type from the qualfied name

Reply:
Yep, will do in next PR.